### DISCUSSION TITLE
Powered by pvlib stickers

### DISCUSSION THREAD

**Discussion by AdamRJensen** [pvlib/pvlib-python#1499] [Ideas]
@pvlib I have been thinking about how neat it would be to have a "powered by pvlib" sticker.

So, if there is support for the idea, I would like to order some "powered by pvlib" stickers that we could bring to PVPMC.

Let me know your thoughts. If positive, then I'd also be interested in which of the two logo types to use and roughly what size would be appropriate.

![image](https://user-images.githubusercontent.com/39184289/180361302-99ea5a07-c7f7-4186-8a0e-f3b6703ee56a.png)

**Comment by mikofski** [comment]
YES! Maybe we can ask NumFOCUS for help?

**Reply by cwhanse** [reply]
How much would they cost? I vote for the left image for the sticker.

**Reply by AdamRJensen** [reply]
I imagine it can't be much :shrug: - in any case I have funds that can cover the costs.

**Reply by AdamRJensen** [reply]
FYI: Stickers have been ordered and should soon be in @mikofski's possession and ready for PVPMC!

**Reply by mikofski** [reply]
I have the stickers!

**Reply by steve-ransome** [reply]
These are great, thanks for giving some out at the PVPMC in Salt Lake. 

We could get photos taken of the stickers on equipment or laptops in use in labs, offices or in the field.

I assume we can use the left hand logo (as that's what's on the stickers) in presentations at conferences?

**Reply by AdamRJensen** [reply]
@steve-ransome You are in fact encouraged to use the logo! :)

It's also stated in the citing section of the documentation: https://pvlib-python.readthedocs.io/en/stable/#citing-pvlib-python
> If you use pvlib-python in a commercial or publicly-available application, please consider displaying one of the “powered by pvlib” logos

**Reply by AdamRJensen** [reply]
Also, if you have any feedback on the sticker quality, sizing, background color, you are encouraged to post it here so we know what to get in case we do this again 😄

**Reply by steve-ransome** [reply]
Thanks Adam, the logo and the stickers look great! 
However when I try to download the logo I can't get the transparent background to work, if I save it as a transparent gif using irfanview I lose the 'powered by' and trying to import it with powerpoint I lose it as well. It seems to have been saved as a png.
I need it for a poster and can use a white background but just in case anyone else has this problem too,

**Reply by AdamRJensen** [reply]
@steve-ransome The image is originally a PNG with transparent background, so if you download it as a .png it should remain transparent. I just tested this locally and it works fine for me when I import it into PowerPoint.

If you have a transparent background in PowerPoint, you can achieve a white background by setting the fill color of the figure to white:
![image](https://user-images.githubusercontent.com/39184289/187963352-6b0a5689-94fc-4255-9e14-64dcf463c747.png)

**Reply by steve-ransome** [reply]
Thanks Adam. What you said didn't work for me through paint or irfanview but what did work was importing the png or gif with the grey solid background (not transparent) into powerpoint then  forcing the grey to be transparent

![image](https://user-images.githubusercontent.com/13416507/187981837-186bd8ac-c880-42e8-813f-ca4b05182788.png)

**Comment by AdamRJensen** [comment]
If you have a cool photo of using the stickers, feel free to showcase them in this post

**Reply by AdamRJensen** [reply]
I used the clear version on my laptop:
<img src="https://user-images.githubusercontent.com/39184289/187743905-be5d0955-fd81-424b-8ec0-1476d35fe0ed.png" width=50% height=50%>
